src/incubator-avatar.html

```diff
@@ -126,6 +164,10 @@
 
           this.abbr = name.split(' ').map((word) => word[0]).join('');
         }
+
+        _displayTooltip() {
```

Comment:
The method's name looks like the method opens the tooltip or something. Consider renaming it, please.